spek-gradle-plugin/build.gradle

```diff
@@ -0,0 +1,48 @@
+apply from: "$rootDir/gradle/common/dependencies.gradle"
```

Comment:
Still not 100% onboard on this. We use Spek and (a little bit) JUnit as well. If I understand correctly the plugin adds `spek` task which will ignore JUnit. BTW, shouldn’t this be a separate PR?